api/python/t4/packages.py

```diff
@@ -226,6 +226,13 @@ def fetch(self, dest=None):
         dest = fix_url(dest)
         copy_file(physical_key, dest, self.meta)
 
+        # return a package reroot package physical keys after the copy operation succeeds
+        # see GH#388 for context
+        entry = self._clone()
+        entry.physical_keys = [dest] + entry.physical_keys[1:]
```

Comment:
I'd just do `= [dest]`; we've never supported multiple physical keys, so no point in pretending.